exercise 5/tables.html

```diff
@@ -0,0 +1,38 @@
+﻿<html>
+    <head>
+        <title>Table Time</title>
+    </head>
+    
+    <body>
+        
+        <table style="border-collapse:collapse;">
+            <thead>
+                <tr>
+                    <th colspan="2" style="color:red">Famous Monsters by Birth Year</th>
+                </tr>
+                <tr style="border-bottom:1px solid black;">
+                    <th style="padding:5px;"><em>Famous Monster</em></th>
```

Comment:
Instead of doing `<em>` here, add the italic in the style attribute, like so:

`<th style="padding: 5px; font-style: italic;">Famous Monster</th>`